cli/Cargo.toml

```diff
@@ -36,6 +37,7 @@ solana-config-program = { path = "../programs/config", version = "1.5.0" }
 solana-faucet = { path = "../faucet", version = "1.5.0" }
 solana-logger = { path = "../logger", version = "1.5.0" }
 solana-net-utils = { path = "../net-utils", version = "1.5.0" }
+solana_rbpf = "=0.1.31"
```

Comment:
fyi, pending PR will require this be bumped to v01.1.32:

https://github.com/solana-labs/solana/pull/12728